includes/functions.php

```diff
@@ -12,7 +12,13 @@ function URLtoPath($url, $baseURL = '', $domainBasedCaching = false)
 
         // query strings are not yet supported so we need to bail is there is one present
```

Comment:
modify this comment so that:
a) we still say that query strings are not supported
b) however we so still remove stage=live from them

Reply:
This section is covered in #120 so I think it should be removed